lib/stat_tracker.rb

```diff
@@ -153,8 +153,8 @@ def games_won_per_team
   end
 
   def game_result?(result, team_id, game_id)
-    game_team = @game_teams.find { |game_team| game_team.game_id == game_id && game_team.team_id == team_id }
-    game_team.team_id == team_id && game_team.result == result
+    game = @game_teams.find { |game_team| game_team.game_id == game_id && game_team.team_id == team_id }
+    game.team_id == team_id && game.result == result
```

Comment:
Since we check this condition, `game.team_id == team_id` ,  in the #find block you don't need it again in line 157!

Reply:
That was the only thing I saw as well! These methods are great!

Reply:
Ah, yes just removed it. Thanks!